packages/logs/README.md

```diff
@@ -117,17 +117,17 @@ window.DD_LOGS.init({
 
-| Parameter             | Type    | Required | Default         | Description                                                                                                    |
-| --------------------- | ------- | -------- | --------------- | -------------------------------------------------------------------------------------------------------------- |
-| `clientToken`         | String  | Yes      |                 | A [Datadog client token][2].                                                                                   |
-| `site`                | String  | Yes      | `datadoghq.com` | The Datadog site of your organization. US: `datadoghq.com`, EU: `datadoghq.eu`                                 |
-| `service`             | String  | No       |                 | The service name for your application.                                                                         |
-| `env`                 | String  | No       |                 | The application’s environment, for example: prod, pre-prod, staging, etc.                                      |
-| `version`             | String  | No       |                 | The application’s version, for example: 1.2.3, 6c44da20, 2020.02.13, etc.                                      |
-| `forwardErrorsToLogs` | Boolean | No       | `true`          | Set to `false` to stop forwarding console.error logs, uncaught exceptions and network errors to Datadog.       |
-| `sampleRate`          | Number  | No       | `100`           | The percentage of sessions to track: `100` for all, `0` for none. Only tracked sessions send logs.             |
-| `silentMultipleInit`  | Boolean | No       |                 | Prevent logging errors while having multiple init.                                                             |
-| `proxyUrl`            | Boolean | No       |                 | Optional proxy URL (ex: https://www.proxy.com/path), see the full [proxy setup guide][6] for more information. |
+| Parameter             | Type    | Required | Default         | Description                                                                                                                           |
+| --------------------- | ------- | -------- | --------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| `clientToken`         | String  | Yes      |                 | A [Datadog client token][2].                                                                                                          |
+| `site`                | String  | Yes      | `datadoghq.com` | The Datadog site of your organization. US: `datadoghq.com`, EU: `datadoghq.eu`                                                        |
+| `service`             | String  | No       |                 | The service name for your application. It should follow the [tag syntax requirements][tag_syntax].                                    |
```

Comment:
This doc is single-sourced for the docs site, so all `[tag_syntax]` links throughout this PR need to be in link list format, `[link][number]`, and the link included at the bottom of the page.